src/event.cpp

```diff
@@ -128,6 +128,7 @@ wcstring event_get_desc(const event_t &e) {
             if (e.param1.pid > 0) {
                 result = format_string(_(L"exit handler for process %d"), e.param1.pid);
             } else {
+                // In events, PGIDs are stored as negative PIDs
```

Comment:
FWIW, This convention predates fish by a couple of decades and was introduced by AT&T System V UNIX. It's why you can call `kill()` with a negative number to send a signal to a process group identified by the absolute value rather than use `killpg()`. My memory is fuzzy but I think `killpg()` was adopted from BSD and introduced in SVR4. Before that the convention was to use the negative of the process group ID.